kzg_prover/src/chips/range/tests.rs

```diff
@@ -130,6 +131,12 @@ impl Circuit<Fp> for TestCircuit {
 
         let add_selector = meta.selector();
 
+        let instance = meta.instance_column();
+        meta.enable_equality(instance);
+
+        let z0 = meta.advice_column();
```

Comment:
Seems like z0 is not used anymore

Reply:
Thanks, I quickly fixed! at [c831f5d](https://github.com/summa-dev/summa-solvency/pull/276/commits/c831f5d2f8e28adb0eed0de593eb3c93ddc0ecd7)